amplipi/ctrl.py

```diff
@@ -1095,7 +1115,6 @@ def exec_stream_command(self, sid: int, cmd: str) -> ApiResponse:
         stream.restart()
       else:
         stream.send_cmd(cmd)
-      self.sync_stream_info()
```

Comment:
Why was this removed?

Reply:
Was not supposed to be removed, put back